src/test/java/org/broadinstitute/consent/http/resources/DarCollectionResourceTest.java

```diff
@@ -289,18 +292,6 @@ void testGetCollectionByReferenceIdNotFound() {
     assertEquals(HttpStatusCodes.STATUS_CODE_NOT_FOUND, response.getStatus());
   }
 
-  @Test
-  void testCancelDarCollection_OKStatus() {
-    DarCollection collection = mockDarCollection();
-    collection.setCreateUserId(researcher.getUserId());
-    when(userService.findUserByEmail(anyString())).thenReturn(researcher);
-    when(darCollectionService.getByReferenceId(any())).thenReturn(collection);
-    initResource();
-
-    Response response = resource.getCollectionByReferenceId(authUser, "1");
-    assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
-  }
```

Comment:
Test removed since the internal logic is replicated by `testGetCollectionByReferenceId()`. Additionally, this test is not testing any cancellation code paths.